Python/compile.c

```diff
@@ -96,6 +96,7 @@ static PySTEntryObject *compiler_symtable_entry(struct compiler *c);
 #define IS_INTERACTIVE(C) compiler_is_interactive(C)
 #define IS_NESTED_SCOPE(C) compiler_is_nested_scope(C)
 #define SCOPE_TYPE(C) compiler_scope_type(C)
+#define QUALNAME(C) compiler_qualname(C)
```

Comment:
I see why you are adding this macro; it fits the pattern.

But OOI, why do we have all these macros in the first place?

Reply:
I don't know why originally, but I think it will be handy for customizing the compiler after I finish the split. We'll see.